clients/ops/admin-ui/src/features/datastore-connections/add-connection/ChooseConnection.tsx

```diff
@@ -38,17 +38,26 @@ const ChooseConnection: React.FC<ChooseConnectionProps> = ({ currentStep }) => {
       250
     )
   );
-
-  useEffect(() => {
-    updateCachedFilters.current(filters);
-  }, [setCachedFilters, filters]);
-
   const { data, isFetching, isLoading, isSuccess } =
     useGetAllConnectionTypesQuery(cachedFilters);
 
   const handleSearchChange = (event: React.ChangeEvent<HTMLInputElement>) =>
```

Comment:
Should we add some constraints here, like debouncing or a minimum search term length before we send anything off to the API?

Reply:
There is a debounce on line 35. I do agree maybe we increase the milliseconds from 250 to 500 and have a minimum character limit of 2. Your thoughts?

Reply:
Have added this to a [follow-up](https://github.com/ethyca/fidesops/issues/1129)